cmd/get/account_password/account_password.go

```diff
@@ -0,0 +1,85 @@
+package account_password
```

Comment:
The package name should be `accountPassword` and similarly, the file name should be `accountPassword.go` as we are following the camelcase for folder structure.